bot/exts/valentines/myvalenstate.py

```diff
@@ -73,8 +69,8 @@ async def myvalenstate(self, ctx: commands.Context, *, name: str = None) -> None
                          " you better"
 
         embed = discord.Embed(
-            title=f'Your Valenstate is {valenstate} \u2764',
-            description=f'{STATES[valenstate]["text"]}',
+            title=f"Your Valenstate is {valenstate} \u2764",
+            description=f"{STATES[valenstate]['text']}",
```

Comment:
This f-string isn't necessary
```suggestion
            description=STATES[valenstate]['text'],
```
Also, can you change line 65 to be
`embed_text = f"You have another match, this being {leftovers[0]}."` to avoid formatting the list directly.

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/e1361d03fac450365d4bdc8d10d8bcf2c99b479f.